src/main/java/com/rcv/Logger.java

```diff
@@ -82,20 +82,27 @@ static void setup() throws IOException {
     Path logPath =
         Paths.get(System.getProperty("user.dir"), EXECUTION_LOG_FILE_NAME).toAbsolutePath();
 
-    // executionHandler writes to the execution log file
-    FileHandler executionHandler =
-        new FileHandler(
-            logPath.toString(), LOG_FILE_MAX_SIZE_BYTES, EXECUTION_LOG_FILE_COUNT, true);
-    executionHandler.setLevel(Level.INFO);
-    logger.addHandler(executionHandler);
+    // executionHandler writes to the execution log file in current working directory
+    try {
+      FileHandler executionHandler =
+          new FileHandler(
+              logPath.toString(), LOG_FILE_MAX_SIZE_BYTES, EXECUTION_LOG_FILE_COUNT, true);
+      executionHandler.setLevel(Level.INFO);
+      logger.addHandler(executionHandler);
+    } catch (IOException exception) {
+      log(Level.WARNING, (String.format("Failed to start system logging!\n" +
```

Comment:
you have an extra set of parens around the call to `String.format`

Reply:
This is removed in the latest commit.